packages/flutter/lib/src/widgets/app.dart

```diff
@@ -422,29 +438,42 @@ class WidgetsApp extends StatefulWidget {
   /// {@endtemplate}
   final Iterable<LocalizationsDelegate<dynamic>> localizationsDelegates;
 
-  /// {@template flutter.widgets.widgetsApp.localeResolutionCallback}
+  /// {@template flutter.widgets.widgetsApp.localeListResolutionCallback}
```

Comment:
Note that I just discovered https://github.com/dart-lang/dartdoc/issues/1812 (our templates are not showing up in the docs for the base classes)